cassdegrees/templates/staff/creation/createsubplan.html

```diff
@@ -94,6 +94,25 @@
                         "to students or be selectable as a requirement elsewhere. Are you sure you want to continue?")) {
                     return false;
                 }
+                // If the user has chosen to publish a subplan but the unit count is inconsistent, verify with the user
+                if (document.getElementById("id_publish").checked) {
+                    var planType = document.getElementById("id_planType").value;
+                    if (planType === "MAJ" && !isValidUnitCount(48) &&
```

Comment:
Is it possible to get this from count units or similar? I'm not 100% sure about putting constants here - could this be extracted elsewhere?